connect-examples/v2/csharp_checkout/Pages/Checkout.cshtml.cs

```diff
@@ -30,19 +31,23 @@ public CheckoutModel( Microsoft.Extensions.Configuration.IConfiguration configur
 
     }
 
-    public IActionResult OnPost()
+    async public Task<IActionResult> OnPost()
     {
       ICheckoutApi checkoutApi = client.CheckoutApi;
       try
       {
+        // Get the currency for the location
+        RetrieveLocationResponse location = await client.LocationsApi.RetrieveLocationAsync(locationId: locationId);
```

Comment:
[Minor] to avoid confusion on `location` vs `Location` how about naming this variable `locationResponse`?

Reply:
Same with `processPayment.chstml.cs`

Reply:
And the examples in other languages as well